consensus/state.go

```diff
@@ -211,6 +219,11 @@ func StateMetrics(metrics *Metrics) StateOption {
 	return func(cs *State) { cs.metrics = metrics }
 }
 
+// StateEventCollector sets the remote event collector.
+func StateEventCollector(ec *remote.Client) StateOption {
```

Comment:
[no change needed] but I find it odd that this file uses the `Set` prefix for some methods and then `State` for other methods that are doing the same thing (i.e. setting a field). IMO

```suggestion
// SetEventCollector sets the remote event collector.
func SetEventCollector(ec *remote.Client) StateOption {
```

makes more sense but I see that `StateMetrics` exists above so this suggestion isn't necessary b/c there isn't consistency w/ respect to names here

Reply:
cool, fixed here to account for the usage as well https://github.com/celestiaorg/celestia-core/pull/970/commits/136931a89999e34c7f7eccc74163f3313920fb93